roofit/roofitcore/test/TestStatistics/testPlot.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   ZW, Zef Wolffs, NIKHEF, zefwolffs@gmail.com
+ *
+ * Copyright (c) 2021, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#include "RooFit.h"
+#include "RooWorkspace.h"
+#include "RooPlot.h"
+#include "RooDataSet.h"
+#include "RooFit/TestStatistics/buildLikelihood.h"
+#include "RooAbsPdf.h"
+#include "RooRealVar.h"
+#include "RooFit/TestStatistics/RooRealL.h"
+#include "RooFit/MultiProcess/Config.h"
+#include "RooMinimizer.h"
+#include <memory>
+
+#include "RooUnitTest.h"
+#include "gtest/gtest.h"
```

Comment:
```suggestion
#include <RooFit.h>
#include <RooWorkspace.h>
#include <RooPlot.h>
#include <RooDataSet.h>
#include <RooFit/TestStatistics/buildLikelihood.h>
#include <RooAbsPdf.h>
#include <RooRealVar.h>
#include <RooFit/TestStatistics/RooRealL.h>
#include <RooFit/MultiProcess/Config.h>
#include <RooMinimizer.h>
#include <RooUnitTest.h>

#include <gtest/gtest.h>

#include <memory>
```
The order of header includes should be:
1. The header corresponding to this implementation (not applicable here)
2. RooFit header
3. Other ROOT headers
4. Other libraries
5. STL headers

Also, just always use `<>` unless you are referring to a relative path, e.g. because the header file is in the same directory (not the case here).